### ISSUE TITLE
Add check for the antivirus software (desktop/mobile)

### ISSUE THREAD

**Issue by strayiker** [brianlovin/security-checklist#5]
This is the most critical part of security!
[popular software](https://www.itproportal.com/guides/the-best-antivirus-software/)

**Comment by brianlovin** [comment]
I'm insanely skeptical of antivirus software; can we pull together some really trustworthy sources on this topic to determine the value? Also we should probably have resources broken down by platform (eg. macos needs may be different from windows)

**Comment by strayiker** [comment]
It's first line of defense against wide range of attacks for most users (mostly non-IT users). It can:
- prevent opening/installation of malicious software/file/web site e.t.c. 
- protect against spy actions (typing, web cam, plugged devices and others.)
- block ad
- protect or notify about OS vulnerabilities
- e.t.c

Most of popular software are cross-platform. E.g:
- nod32: win, mac, linux, mobile
- kaspersky: win, mac, linux, mobile
- avg: win, mac, linux?, mobile
- avast: win, mac, linux, mobile
- avira: win, mac, mobile

For me antivirus it's in 99.9% cases useless too. But it obviously can save others.
Idk any trustworthy sources about it, sorry.

**Comment by brandonweeks** [comment]
It is essentially impossible for AV software to add value on mobile platforms, since the APIs to introspect the system meaningfully don't exist for security reasons.

Windows Defender is arguably the best option for Windows. 

macOS has Gatekeeper, which when enabled should be far more effective than AV.

I doubt Linux AV software is detecting anything other than Windows viruses, alas I don't have any data to back that up.

**Comment by dmleong** [comment]
I would be curious to see the benefits of an antivirus software versus browser extensions like Privacy Badger and UBlock. Does anyone have any resources comparing these?

**Comment by strayiker** [comment]
Here are some of the features that my AV provides (windows):
- Camera protection: shows me dialog with option to block access if any program (including OS) try to use camera. Now i know that win 10 trying to shot me sometimes 😱.
- Updates my software.
- Blocks ads and trackers - not so good as `AdBlock`/`UBlock` and `DuckDuckGo Privacy Essentials` but can catch what this extensions are missed.
- Whenever I go to a payment page or a bank site, opens this page inside a new protected browser window. Disables all extensions, checks for rootkits, OS vulnerabilities and site certificates.
- Removes offers to install additional software from installation software.

**Comment by dmleong** [comment]
Thanks @strayiker. I think this is an interesting case of less technical users versus more technical users! Seems like having all of these benefits in one spot is useful for folks who just want to install one thing instead of a multitude of browser extensions. I'm in favor of making it easy for people to be safer 👍

**Comment by brandonweeks** [comment]
Less technical users should be steered towards platforms that have secure defaults, such as:
- Chrome OS
- iOS
- macOS with Gatekeeper
- Windows 10 in S mode

Using Windows with third party antivirus software has been common advice for decades and it isn't working. The app store model pioneered by Apple has proven to be a far better alternative for most users.

**Comment by daniel-vera-g** [comment]
> I would be curious to see the benefits of an antivirus software versus browser extensions like Privacy Badger and UBlock. Does anyone have any resources comparing these?

Antivirus software sometimes(sometimes just spam) keep your OS safe. Browser Extensions help you mostly with unwanted javascript, cookies, ads... In a nutshell: 
1. Antivirus -> OS(OS Scurity)
2. Browser Extensions -> Keeping your data safe from websites on the net.

You can live without an Antivirus if you don't do stupid things. But nowadays surfing the web without some basic extensions gets really nasty.

**Comment by daniel-vera-g** [comment]
> Less technical users should be steered towards platforms that have secure defaults, such as:
> 
> * Chrome OS
> * iOS
> * macOS with Gatekeeper
> * Windows 10 in S mode
> 
> Using Windows with third party antivirus software has been common advice for decades and it isn't working. The app store model pioneered by Apple has proven to be a far better alternative for most users.

If your don't need software like Photoshop or Office, you can also try some User friendly linux distros. I've been having less problems with Linux than with windows for all this time. And in comparison to the past, there are a lot of good software also avalaible for Linux.

**Comment by jgallias** [comment]
I have to agree with @brandonweeks here, the biggest issue people have on Windows/macOS is just not doing their updates. There are viable alternative platforms now that won't let people shoot themselves in the foot.
